MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -105,6 +128,20 @@ - (void)deleteAppMetadata:(MSIDAppMetadataCacheItem *)appMetadata
     }
 }
 
+- (void)deleteKey:(MSALTestAppAsymmetricKey *)key
+{
+    NSDictionary *query = [NSDictionary dictionaryWithObjectsAndKeys:
```

Comment:
Should we pass keychain group in the query as well?

Reply:
The code used to delete the key in MSIDAssymetricKeyKeychainGenerator adds the kechain access group to the query by default so we should be good here.